.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,8 +1,8 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: 'Your Title'
-labels: ["type: new", "type: bug"]
+title: Your Title
+labels: 'type: bug, type: new'
```

Comment:
Passing by, maybe we could remove the `type: new` flag.
It was an attempt and we do not use it very much, actually.